packages/profile-sync-controller/src/sdk/authentication.ts

```diff
@@ -50,6 +58,36 @@ export class JwtBearerAuth implements SIWEInterface, SRPInterface {
     return await this.#sdk.signMessage(message);
   }
 
+  async pairIdentifiers(pairing: Pair[]): Promise<void> {
```

Comment:
Hmmm...

I'm hesitant if implementations should be done in this combined class vs in the individual implementations.

Context:
- Combined Class: 1 class; but poor type interface
- Individual Class: the developer can choose a specific implementation & has great type interface.

Reply:
Yes lets move the implementation to the individual classes, that way the better typed implementations can also utilise this method.

Reply:
Only annoying thing is that the implementation is identical on both implementations, so a bit redundant. (Sadly JS does not support the trait system lol).

Even though there would be duplication, I think it is nicer to have that abstraction.